.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "6.2"
+  - "6.10.3"
```

Comment:
Honestly, `6.0` may be the best version here, although in general we shouldn't expect language support to change between minor releases of Node versions, so specifying any version of Node 6 should presumably be sufficient. Maybe `6` is sufficient?